pytest_mpl/plugin.py

```diff
@@ -196,6 +204,12 @@ def pytest_configure(config):
         if results_dir is not None:
             results_dir = os.path.abspath(results_dir)
 
+        default_style = (config.getoption("--mpl-default-style") or
+                         config.getini("mpl-default-style"))
+
+        default_tolerance = (config.getoption("--mpl-default-tolerance") or
+                             config.getini("mpl-default-tolerance"))
```

Comment:
The options will return strings so need to cast to float:
```suggestion
        default_tolerance = float(config.getoption("--mpl-default-tolerance") or
                                  config.getini("mpl-default-tolerance") or
                                  "2")
```